pages/overview.mdx

```diff
@@ -0,0 +1,47 @@
+import { PortalUSP } from 'components/pop'
+
+## What is the Portal Network?
+
+<br />
+<br />
+<PortalUSP />
+<br />
+
+Ethereum users have two options: 
+
+- dedicate a non-trivial amount of computing resources and bandwidth to running an Ethereum node
+- trust a centralized third-party (such as an RPC provider) to provide Ethereum data
+
+The Portal Network introduces a third option for Ethereum users: 
+- access Ethereum data in a decentralized way with minimal hardware and bandwidth requirements.
+
+The Portal Network is a peer-to-peer protocol that runs parallel to Ethereum. Ethereum data is **distributed across the Portal Network**, instead of being copied in every individual node. 
+This allows users to access Ethereum data with minimal hardware and bandwidth requirements and almost instant syncing.
+
+
+## How does Portal work?
+
+The Portal Network is actually several peer-to-peer networks (state, beacon, history, transaction gossip and canonical transaction index networks) that can be accessed using a Portal Network [client](../clients). 
+Each of these networks stores a specific subset of the data stored by an Ethereum full node. 
+Each individual Portal client only stores a tiny fraction of each type of data. 
+However, the Portal Network (the complete set of nodes) stores **all** the historical Ethereum data spanning from genesis right up to within a block or two of the head of the chain.
+
+When you request data from your Ethereum node, you are looking up some specific information in your local copy of the blockchain. If you request data through an RPC provider,
+you look it up in someone else's stored copy of the blockchain (and they may manipulate it or track you as they handle the request and response). When you request data from your Portal node, 
+the request is broadcast across the network from peer to peer, until it eventually reaches the node storing the specific information you want. This node then returns the response to you. 
+There are no intermediaries, and no individual participants need to store large amounts of chain data or handle large numbers of requests.
+
+## Why is Portal Network important?
+
+Syncing Ethereum full nodes involves downloading hundreds of gigabytes of history and state data, verifying it and storing it locally. 
+This requires the node operator to dedicate substantial CPU, network bandwidth, hard disk storage and RAM in order to run the node. This
```

Comment:
```suggestion
This requires the node operator to dedicate substantial CPU, network bandwidth (1TB per month), hard disk storage (1+ TB of SSD) and RAM in order to run the node. This
```

Reply:
I dunno if this improves this part at all, but I wanted to see what it was like to add some specificity to the amounts here.  take it or leave it on your best judgement.